tests/src/HighlightController.cpp

```diff
@@ -224,6 +224,17 @@ class MockHelix : public IHelix
                  ResultCallback<std::vector<HelixChannelEmote>> successCallback,
                  HelixFailureCallback failureCallback),
                 (override));
+    
+    MOCK_METHOD(void, getGlobalBadges,
+                (ResultCallback<HelixGlobalBadges> successCallback,
+                 FailureCallback<HelixGetGlobalBadgesError, QString> failureCallback),
+                (override));
+
+    MOCK_METHOD(void, getChannelBadges,
+                (QString broadcasterID,
+                 ResultCallback<HelixChannelBadges> successCallback,
+                 FailureCallback<HelixGetChannelBadgesError, QString> failureCallback),
+                (override));
```

Comment:
```suggestion
    // The extra parenthesis around the failure callback is because its type contains a comma
    MOCK_METHOD(void, getGlobalBadges,
                (ResultCallback<HelixGlobalBadges> successCallback,
                 (FailureCallback<HelixGetGlobalBadgesError, QString> failureCallback)),
                (override));

    // The extra parenthesis around the failure callback is because its type contains a comma
    MOCK_METHOD(void, getChannelBadges,
                (QString broadcasterID,
                 ResultCallback<HelixChannelBadges> successCallback,
                 (FailureCallback<HelixGetChannelBadgesError, QString> failureCallback)),
                (override));
```

Reply:
Thanks for the correction!